inference-engine/tests/functional/plugin/cpu/single_layer_tests/broadcast.cpp

```diff
@@ -0,0 +1,375 @@
+                                                               axesMappingOp,
+                                                               mode);
+        } else if (mode == ov::op::BroadcastType::NUMPY) {
+            if (isTargetShapeConst) {
+                auto targetShapeConst = ov::op::v0::Constant::create(ov::element::i64, {targetShapeRank}, targetShape);
+                broadcastOp = std::make_shared<ov::op::v3::Broadcast>(paramOuts[0],
+                                                                      targetShapeConst,
+                                                                      mode);
+            } else {
+                broadcastOp = std::make_shared<ov::op::v3::Broadcast>(paramOuts[0],
+                                                                      paramOuts[1],
+                                                                      mode);
+            }
+        }
+
+        broadcastOp->get_rt_info() = getCPUInfo();
+        function = makeNgraphFunction(netPrecision, functionParams, broadcastOp, "Broadcast");
+    }
+
+    void generate_inputs(const std::vector<ngraph::Shape>& targetInputStaticShapes) override {
+        inputs.clear();
+        const auto& funcInputs = function->inputs();
+        for (size_t i = 0lu; i < funcInputs.size(); i++) {
+            const auto& funcInput = funcInputs[i];
+            ov::runtime::Tensor tensor;
+            if (funcInput.get_node()->get_friendly_name() == "targetShape") {
+                tensor = ov::runtime::Tensor{ov::element::i64, targetInputStaticShapes[i]};
+                auto data = tensor.data<ov::element_type_traits<ov::element::i64>::value_type>();
+                for (size_t i = 0lu; i < targetShape.size(); i++) {
+                    data[i] = targetShape[i];
+                }
+            } else if (funcInput.get_node()->get_friendly_name() == "axesMapping") {
+                tensor = ov::runtime::Tensor{ov::element::i64, targetInputStaticShapes[i]};
+                auto data = tensor.data<ov::element_type_traits<ov::element::i64>::value_type>();
+                for (size_t i = 0lu; i < axesMapping.size(); i++) {
+                    data[i] = axesMapping[i];
+                }
+            } else {
+                if (funcInput.get_element_type().is_real()) {
+                    tensor = ov::test::utils::create_and_fill_tensor(
+                        funcInput.get_element_type(), targetInputStaticShapes[i], 10, 0, 1000);
+                } else {
+                    tensor = ov::test::utils::create_and_fill_tensor(funcInput.get_element_type(), targetInputStaticShapes[i]);
+                }
+            }
+            inputs.insert({funcInput.get_node_shared_ptr(), tensor});
+        }
+    }
+
+    std::vector<int64_t> targetShape, axesMapping;
+};
+
+TEST_P(BroadcastLayerCPUTest, CompareWithRefs) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
+
+    run();
+//    CheckPluginRelatedResults(executableNetwork, "Broadcast");
```

Comment:
Please uncomment this call. All the necessary changes are in master now.

Reply:
Done.